apps/storybook/src/forms/ToggleGroup.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { ToggleGroup, ToggleGroupItem } from "@optiaxiom/react/unstable";
+import { IconLayoutGrid } from "@tabler/icons-react";
+
+export default {
+  argTypes: {
+    onClick: { action: "click" },
+  },
+  component: ToggleGroup,
+} as Meta<typeof ToggleGroup>;
+
+type Story = StoryObj<typeof ToggleGroup>;
+
+export const Basic: Story = {
+  args: {
+    children: (
+      <>
+        <ToggleGroupItem icon={<IconLayoutGrid />} value="a" />
+        <ToggleGroupItem icon={<IconLayoutGrid />} value="b" />
+        <ToggleGroupItem icon={<IconLayoutGrid />} value="c" />
```

Comment:
<img width="426" alt="Screenshot 2024-09-30 at 3 13 42 PM" src="https://github.com/user-attachments/assets/a0697109-1946-40fd-bb45-8ed243362d86">

let's use one of these two as an example here - rather than just 3 identical grid icons

Reply:
This seems more applicable. Will update it bhai.
Thanks!